tests/utils/test_archive.py

```diff
@@ -136,6 +129,27 @@ def test_archive02(self):
                 filename = "{0}_0.zip".format(timestamp)
             self.assertTrue(filename in archived_entries, "'{0}' is not in the list of unpurged entires".format(filename))
 
+    def test_purge_legacy_goal_state_history(self):
+        with patch("azurelinuxagent.common.conf.get_lib_dir", return_value=self.tmp_dir):
+            legacy_files = [
+                'GoalState.1.xml',
+                'VmSettings.1.json',
+                'Prod.1.manifest.xml',
+                'ExtensionsConfig.1.xml',
+                'Microsoft.Azure.Extensions.CustomScript.1.xml',
+                'SharedConfig.xml',
+                'HostingEnvironmentConfig.xml',
+                'RemoteAccess.xml'
+            ]
+            legacy_files = [os.path.join(self.tmp_dir, file) for file in legacy_files]
```

Comment:
Looks like using the variable `file` is tripping up pylint on python2.7. Wanted to save you having to look for it.

Reply:
thanks; i fixed the instances below this one, will fix it too